doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -381,6 +422,51 @@ There are three situations:
     Check (b 0).
     Unset Printing Coercions.
 
+.. _example-reverse-coercion:
+
+.. example:: Reverse coercions.
+
+  .. coqtop:: in
+
+    Structure S := {
+      ssort :> Type;
+      sstuff : ssort;
+    }.
+    Definition test (s : S) := sstuff s.
+    Canonical Structure S_nat := {| ssort := nat; sstuff := 0; |}.
+
+  .. coqtop:: all
+
+    Check test (nat : Type).
+
+.. _example-reverse-coercion-attribute:
+
+.. example:: Reverse coercions (reverse attribute).
+
+  .. coqtop:: in
+
+    Structure S' := {
+      ssort' : Type;
+      sstuff' : ssort';
+    }.
+    Coercion ssort' : S' >-> Sortclass.
+    Definition test' (s : S') := sstuff' s.
+    Canonical Structure S_nat' := {| ssort' := nat; sstuff' := 0; |}.
+
+  This is the same as the previous example, except that, by default,
+  the reverse attribute in unset:
```

Comment:
```suggestion
  Since there's no `:>` on the definition of `ssort'`, the :attr:`reverse` attribute is not set:
```

Don't need to say "no :>" in two places, take your pick.